### ISSUE TITLE
Postgres introspect relation does not exist

### ISSUE THREAD

**Issue by fermentfan** [drizzle-team/drizzle-kit-mirror#130]
In two different Prisma projects I maintain I get the same error trying out the introspect feature. It fails with the following error:

```bash
npx drizzle-kit introspect:pg
drizzle-kit: v0.19.10
drizzle-orm: v0.27.2

No config path provided, using default 'drizzle.config.ts'
Reading config file '/xxx/drizzle.config.ts'
[✓] 13 tables fetched
[⣷] 0  columns fetching
[⣷] 0  enums fetching
[⣷] 0  indexes fetching
[⣷] 0  foreign keys fetching
error: relation "public.<RELATION NAME REDACTED>" does not exist
    at Parser.parseErrorMessage (/xxx/node_modules/drizzle-kit/index.cjs:40646:98)
    at Parser.handlePacket (/xxx/node_modules/drizzle-kit/index.cjs:40487:25)
    at Parser.parse (/xxx/node_modules/drizzle-kit/index.cjs:40411:34)
    at Socket.<anonymous> (/xxx/node_modules/drizzle-kit/index.cjs:40687:44)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 125,
  severity: 'ERROR',
  code: '42P01',
  detail: undefined,
  hint: undefined,
  position: '1230',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'namespace.c',
  line: '427',
  routine: 'RangeVarGetRelidExtended'
```

I redacted the name for the relation but I can go into detail if it's needed. The relation name is different in the other project of course. Sadly I have a hard time debugging this. Any tips? I have the feeling it has to do with case sensitivity...

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.11`

**Comment by fermentfan** [comment]
Yay amazing! Thank you for the fix. I just tried it out and it works.
